### DISCUSSION TITLE
Request: Automatically color table cells

### DISCUSSION THREAD

**Discussion by NickLarsenNZ** [grafana/grafana#56774] [Dashboards, Panels & Visualizations]
I'm displaying Kubernetes Pod logs in a table, and would like to clearly show when the Pod name (uniquely generated) changes.

I've drawn a line where the pod name changes. Without that line it is quite difficult to see.

![image](https://user-images.githubusercontent.com/10092581/195287726-843f8b07-b4fa-4f34-b6e2-a52a3aa0e5e9.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
